mobile/src/components/common/button/button.tsx

```diff
@@ -29,15 +39,19 @@ const Button: FC<Props> = ({ label, icon, variant, onPress }) => {
           height={20}
         />
       )}
-      <Text
-        style={{
-          ...styles.label,
-          ...styles[`button${variant}Label`],
-          marginLeft: textMarginLeft,
-        }}
-      >
-        {label}
-      </Text>
+      {!isLoading ? (
```

Comment:
It's not user friendly to hide label when loading, it's better to wrap label and loader in View and make loader absolute, and also don't forget about horizontal padding for button in this case